ignite/metrics/gan/utils.py

```diff
@@ -19,12 +19,13 @@ class InceptionModel(torch.nn.Module):
 
     def __init__(self, return_features: bool, device: Union[str, torch.device] = "cpu") -> None:
         try:
+            import torchvision
             from torchvision import models
         except ImportError:
             raise RuntimeError("This module requires torchvision to be installed.")
         super(InceptionModel, self).__init__()
         self._device = device
-        if Version(torch.__version__) <= Version("1.7.0"):
+        if Version(torchvision.__version__) < Version("0.13.0"):
```

Comment:
@puhuk please provide a link here in the comments on which version `models.Inception_V3_Weights.DEFAULT` appeared to make sure that we are doing correct check.
Same for below. Thanks

Reply:
`pretrained` is deprecated since 0.13 and will be removed in 0.15
https://github.com/pytorch/vision/blob/0eb8aabd9fd5950d3867dc8e6b6b81006c8d7416/torchvision/models/_utils.py#L209-L210